### PR TITLE
sort libraries by last activity at

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster https://app.asana.com/0/32728208409723/93569536157282

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala [hunk 1]
```diff
@@ -568,7 +568,7 @@ class KeepRepoImpl @Inject() (
   // Make compiled in Slick 2.1
 
   def getByLibrary(libraryId: Id[Library], offset: Int, limit: Int, excludeSet: Set[State[Keep]])(implicit session: RSession): Seq[Keep] = {
-    (for (b <- rows if b.libraryId === libraryId && !b.state.inSet(excludeSet)) yield b).sortBy(r => (r.keptAt desc, r.id desc)).drop(offset).take(limit).list
+    (for (b <- rows if b.libraryId === libraryId && !b.state.inSet(excludeSet)) yield b).sortBy(r => (r.lastActivityAt desc, r.id desc)).drop(offset).take(limit).list
   }
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala:572]: We should not use this method anymore, `Keep.libraryId` may be removed going forward.

**Comment by camhashemi** [comment]
@ryanpbrewster updated

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala [hunk 2]
```diff
@@ -79,16 +79,9 @@ class LibraryInfoCommanderImpl @Inject() (
...
-      val oldWay = keepRepo.getByLibrary(libraryId, offset, limit)
-      val newWay = ktlRepo.getByLibraryIdSorted(libraryId, Offset(offset), Limit(limit)) |> keepCommander.idsToKeeps
-      if (newWay.map(_.id.get) != oldWay.map(_.id.get)) {
-        log.info(s"[KTL-MATCH] getKeeps($libraryId, $offset, $limit): ${newWay.map(_.id.get)} != ${oldWay.map(_.id.get)}")
-      }
-      if (useMultilibLogic) newWay else oldWay
-    }
-    else Future.successful(Seq.empty)
+  def getKeeps(libraryId: Id[Library], offset: Int, limit: Int): Seq[Keep] = {
+    if (limit > 0) db.readOnlyReplica(implicit s => ktlRepo.getByLibraryIdSorted(libraryId, Offset(offset), Limit(limit)) |> keepCommander.idsToKeeps)
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:83]: Why do we even test for this? It seems like such a weird "optimization"?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:83]: /shrug

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:83]: I mean, why open a db session if you don't have to?

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala:572]: it's used everywhere, so I'll just update it and not use it for library keeps, but one day we will need to migrate it.

**Pull request merged**